### DISCUSSION TITLE
Mulighet til å legge til egne fargetemaer

### DISCUSSION THREAD

**Discussion by PeterRydberg** [entur/tavla#312] [Ideas]
Denne er ganske selvforklarende. Jeg tror ikke det er nødvendig med så veldig komplisert design, men en knapp som åpner en modal fra temasiden gir mening. Herfra kan man få en liste med alle variablene man kan endre (og fargepreview), og så setter man én og én hex-variabel der man vil. Det er snakk om oppimot 30 ulike farger, så det kan bli vanskelig å få det til å se fancy ut. Da blir det fort for litt spesielt interesserte å legge til egne temaer, men jeg tror det er nyttig å ha muligheten!

Til ettertanke burde disse fargetemaene også kobles til en bruker i likhet med #229, slik at man slipper å lage dem på nytt for hver tavle.

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
